### PR TITLE
[16.0][mig] base_menu_visibility_restriction: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Review by kikopeiro — APPROVED** [review]
Functional review / no code review. :+1:  thanks !

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM

File: base_menu_visibility_restriction/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
 * `Tecnativa <https://www.tecnativa.com>`_:
 
   * Víctor Martínez
+  * Dhara Solanki <dhara.solanki@initos.com>
```
Comment (pedrobaeza) [inline base_menu_visibility_restriction/readme/CONTRIBUTORS.rst]: Please put this deindented one level for not being inside Tecnativa

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**